### ISSUE TITLE
extension options on FYPO

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#1294]
spotted this last night:
Get 2 transcript options (slightly different flavours) for this term
![2x extension](https://cloud.githubusercontent.com/assets/7359272/20833096/b497b91a-b885-11e6-8127-bafaf3aeb462.jpg)

I will see if I can  figure if its a config issue later....

**Comment by mah11** [comment]
Fixed the summary, as the screenshot shows a FYPO term.

This can happen when a term has both user and admin extensions available that use the same display text. It's particularly likely to crop up with RNA-related FYPO terms because I've had to do a bit of juggling to deal with LTRs (we're waiting for an "LTR transcript" SO term to be able to improve the config, and in the meantime LTRs are admin-only because technically the currently available SO terms aren't optimal for the FYPO extension purpose).

For this specific case I think I may be able to fiddle with the config so only one "assayed transcript" option will show up even for us, but if it doesn't work out, at least users will still only see the one.
